ubuntu-frame/snap/local/bin/daemon-wrapper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+DAEMON_ARGUMENTS=$(snapctl get camera-detect-stream.arguments)
+
+if [ -z "${DAEMON_ARGUMENTS}" ]; then
+    exec $@
+else
+    exec $@ $DAEMON_ARGUMENTS
+fi
```

Comment:
Does it make a difference if exec is called with an empty `DAEMON_ARGUMENTS`?

Reply:
I had quotes `"` around the variables before, and in that case the space confuse the python interpreter. I had to remove the quotes anyway because otherwise python sees the filename and arguments all together as the filename. So that also fixed this. Now it's simplified.